api/src/Controller/Event/DeleteEvent.php

```diff
@@ -47,9 +47,11 @@ public function buildResource(Request $request, Response $response, $params): ar
         $permissions = ['api.delete.event'];
         $this->checkPermissions($permissions);
         $this->getEvent($params['id']);
```

Comment:
I think we can probably remove this line on 49, right? Or is it the case that the caller might pass in "current" and we do need to find the ID before deleting? It seems like we're only using the parameters.